x/monitoringp/simulation/simulation.go

```diff
@@ -0,0 +1 @@
+package simulation
```

Comment:
Why creating this file?

Reply:
Just to make things symmetric with the other packages.  At some point, I assume we will actually create the simulations, so this is a placeholder.